app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,20 @@
+class AdminController < ApplicationController
+  def index
+    @shelters = Shelter.order_by_name_reverse
+    @pendingshelters = Shelter.shelters_apps_in_progress
+  end
+
+  def show
+    @app = App.find(params[:id])
+  end
+
+  def update
+    @app = App.find(params[:id])
+    if params[:approved] == "true"
+      @app.update!(status: 2)
```

Comment:
Great work using enums for status!

Reply:
For readability, you can use the string or symbol version of the enum instead of the integer and Rails is smart enough to map it to the integer when it gets to the database